collect_app/src/test/java/org/odk/collect/android/preferences/JsonPreferencesGeneratorTest.kt

```diff
@@ -66,4 +67,63 @@ class JsonPreferencesGeneratorTest : TestCase() {
         assertThat(jsonPrefs, not(containsString("password")))
         assertThat(jsonPrefs, not(containsString("123456")))
     }
+
+    @Test
+    fun `Only saved settings should be included in json`() {
+        val generalPrefs = buildMap<String, Any> {
+            put(GeneralKeys.KEY_DELETE_AFTER_SEND, true)
+            put(GeneralKeys.KEY_APP_THEME, "dark_theme")
+        }
+
+        val adminPrefs = buildMap<String, Any> {
```

Comment:
You can just use `mapOf` here instead to avoid the need for the new experimental API:

```kotlin
mapOf(AdminKeys.KEY_GET_BLANK to false, AdminKeys.KEY_DELETE_SAVED to false)
```

Reply:
Right, fixed.